node/src/Transaction.ts

```diff
@@ -5,7 +5,8 @@
 import {
     BaseClient, // eslint-disable-line @typescript-eslint/no-unused-vars
     GlideString,
-    ReadFrom, // eslint-disable-line @typescript-eslint/no-unused-vars
+    HashDataType,
+    convertFieldsAndValuesForHset, // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
You need to suppress eslint for imports only if you're using imported symbol in docs only.
No need to suppress if you're using in your code.
```suggestion
    convertFieldsAndValuesForHset,
```